src/Sentry/SentryOptionsExtensions.cs

```diff
@@ -236,5 +238,25 @@ internal static void SetupLogging(this SentryOptions options)
                 options.DiagnosticLogger = null;
             }
         }
+
+        internal static string? TryGetDsnSpecificCacheDirectoryPath(this SentryOptions options)
+        {
+            if (string.IsNullOrWhiteSpace(options.CacheDirectoryPath))
+            {
+                return null;
```

Comment:
Perhaps at the call site but please log out the path it resolved to or if there was no caching at all